kubernetes/assets/recommended_monitors/monitor_pods_failed_state.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "[kubernetes] Monitor Kubernetes Failed Pods in Namespaces",
+	"type": "query alert",
+	"query": "change(avg(last_5m),last_5m):sum:kubernetes_state.pod.status_phase{phase:failed} by {kubernetes_cluster,kube_namespace} > 10",
```

Comment:
threshold is completely opinionated. Maybe mention that this is highly variable in the monitor message.